runtime/src/main/scala/akka/grpc/internal/Grpc.scala

```diff
@@ -0,0 +1,109 @@
+package akka.grpc.internal
+
+import akka.annotation.InternalApi
+import akka.grpc.{ Codec, GrpcProtocol, Identity }
+import akka.grpc.GrpcProtocol.{ Frame, GrpcProtocolReader, GrpcProtocolWriter }
+import akka.http.scaladsl.model.{ ContentType, MediaType }
+import akka.http.scaladsl.model.HttpEntity.ChunkStreamPart
+import akka.stream.impl.io.ByteStringParser.{ ByteReader, ParseResult, ParseStep }
+import akka.stream.impl.io.ByteStringParser
+import akka.stream.stage.GraphStageLogic
+import akka.stream.Attributes
+import akka.stream.scaladsl.Flow
+import akka.util.ByteString
+import akka.NotUsed
+import io.grpc.{ Status, StatusException }
+
+/**
+ * Functionality common to all gRPC protocols.
+ */
+@InternalApi
+object Grpc {
```

Comment:
I dislike this object name as it is too generic. Should it be `GrpcProtocolHelpers`? It mostly is a companion to `AbstractGrpcProtocol`, right?

Reply:
It is effectively the companion to `AbstractGrpcProtocol` yes. Happy to make it formal if you concur.

Reply:
Yes, that would be great.

Reply:
I think this is the last bit, so I'll pick it up and merge! Almost there!